### PR TITLE
update changelog version to 1.3.2

### PR DESCRIPTION
**Pull request by entonbiba**
updated version and changelog, to be updated once #984 #983 #982 get merged

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "wavesurfer.js",
-  "version": "1.3.1",
+  "version": "1.3.2",
   "homepage": "http://wavesurfer-js.org",
```
Comment (thijstriemstra) [inline bower.json]: not related but should be https (also in package.json)

**Comment by thijstriemstra** [comment]
I can't put out releases, and since the version nr was bumped (that I usually leave up to @katspaugh) it's probably best @katspaugh merges and releases this PR.

**Comment by thijstriemstra** [comment]
No, nevermind. The dates you commited are already out of date @entonbiba. Let's replace the version and date in top changelog section to 'unreleased' or something and revert the version nr change. Sorry.

**Comment by entonbiba** [comment]
@thijstriemstra ok I'll update the version and date in latest once the remaining pull requests get merged, closing this one.

**Pull request closed without merging**